locksmith/src/operations/wedlocksOperations.ts

```diff
@@ -87,10 +86,14 @@ export const notifyNewKeysToWedlocks = async (
  * @param key
  */
 export const notifyNewKeyToWedlocks = async (key: any, network?: number) => {
+  const lockAddress = key.lock.address
+  const ownerAddress = key.owner.address
+  const tokenId = key?.keyId
```

Comment:
Not sure we need `?` here, or we need it everywhere above as well

Reply:
`keyId` is not always passed as parameter, i update the `key: any` type definition to: 

```
 key: {
    lock: {
      address: string
      name?: string
    }
    owner: {
      address: string
    }
    keyId?: string
  },
  ```